documentation/slack-webhook-integration.md

```diff
@@ -0,0 +1,18 @@
+
+### Step 1: Create a Webhook in Slack
```

Comment:
I don't think we should use this technique as it's not supported by Slack anymore.
This is the official way: https://github.com/DFE-Digital/technical-guidance/pull/183